Source/SaveExtension/Private/Serialization/SlotDataTask_Saver.cpp

```diff
@@ -110,6 +110,16 @@ void USlotDataTask_Saver::OnStart()
 	Finish(false);
 }
 
+void USlotDataTask_Saver::Tick(float DeltaTime)
+{
+	Super::Tick(DeltaTime);
+
+	if (SaveInfoTask->IsDone() && SaveDataTask->IsDone())
```

Comment:
I would check the validity of this pointers, appart from that, good job finding the issue!

Reply:
done